### PR TITLE
[CSL-730] Catch&ignore exceptions from endPointClosed

### PR DESCRIPTION
**Pull request by volhovM**
https://issues.serokell.io/issue/CSL-730
cc @georgeee

### CODE REVIEW

File: src/Node/Internal.hs [hunk 1]
```diff
@@ -623,7 +623,8 @@ nodeDispatcher node handlerIn handlerInOut =
           NT.ConnectionClosed connid -> connectionClosed state connid >>= loop
 
           -- When the end point closes, we're done.
-          NT.EndPointClosed -> endPointClosed state
+          NT.EndPointClosed -> 
+              endPointClosed state `catch` (\(e :: SomeException) -> pure ())
```
Comment (georgeee) [inline src/Node/Internal.hs]: Could you send errors to logDebug instead of just ignoring?

**Comment by volhovM** [comment]
I've removed two commits because the same thing was implemented in the other branch. Though having `shell.nix` would be good.

**Comment by avieth** [comment]
LGTM

**Comment by avieth** [comment]
Looks like most of this was already merged, separately. Do we still need the remaining changes?

**Comment by volhovM** [comment]
This was fixed already, PR is obsolete.

**Pull request closed without merging**